### ISSUE TITLE
Delay downloading symbol files after rebooting after upgrading

### ISSUE THREAD

**Issue by jyxjjj** [valinet/ExplorerPatcher#3516]
After a reboot due to upgrading, seems symbol files should be update and re-download.
It starts downloading when boot ended immediately.
Due to the network service doesn't started successfully when boot ended, I think it should have a delay.
Such as 30 seconds or 1 minute.

**Comment by pyrates999** [comment]
Windows actually has api calls that applications can automatically check if the internet is available.  So a delay here is not necessary.

**Comment by jyxjjj** [comment]
> Windows actually has api calls that applications can automatically check if the internet is available.  So a delay here is not necessary.

Yeah that's okay too, but will needs to check more times, i think delay is better
